bitsail-common/src/main/java/com/bytedance/bitsail/common/option/ReaderOptions.java

```diff
@@ -64,47 +66,138 @@ public interface ReaderOptions {
    * ]
    */
   ConfigOption<List<Map<String, Object>>> READER_CONFIG_LIST =
-      ConfigOptions.key(READER_PREFIX + "reader_conf_list")
+      key(READER_PREFIX + "reader_conf_list")
           .onlyReference(new TypeReference<List<Map<String, Object>>>() {
           });
 
   interface BaseReaderOptions {
 
     ConfigOption<List<ColumnInfo>> COLUMNS =
-        ConfigOptions.key(READER_PREFIX + "columns")
+        key(READER_PREFIX + "columns")
             .onlyReference(new TypeReference<List<ColumnInfo>>() {
             });
 
     ConfigOption<Integer> READER_PARALLELISM_NUM =
-        ConfigOptions.key(READER_PREFIX + "reader_parallelism_num")
+        key(READER_PREFIX + "reader_parallelism_num")
             .noDefaultValue(Integer.class);
 
     ConfigOption<String> CHARSET_NAME =
-        ConfigOptions.key(READER_PREFIX + "charset_name")
+        key(READER_PREFIX + "charset_name")
             .defaultValue("");
 
     ConfigOption<String> DB_NAME =
-        ConfigOptions.key(READER_PREFIX + "db_name")
+        key(READER_PREFIX + "db_name")
             .noDefaultValue(String.class);
 
     ConfigOption<String> TABLE_NAME =
-        ConfigOptions.key(READER_PREFIX + "table_name")
+        key(READER_PREFIX + "table_name")
             .noDefaultValue(String.class);
 
     ConfigOption<String> PARTITION =
-        ConfigOptions.key(READER_PREFIX + "partition")
+        key(READER_PREFIX + "partition")
             .noDefaultValue(String.class);
 
     ConfigOption<String> USER_NAME =
-        ConfigOptions.key(READER_PREFIX + "user_name")
+        key(READER_PREFIX + "user_name")
             .noDefaultValue(String.class);
 
     ConfigOption<String> PASSWORD =
-        ConfigOptions.key(READER_PREFIX + "password")
+        key(READER_PREFIX + "password")
             .noDefaultValue(String.class);
 
     ConfigOption<String> CONTENT_TYPE =
-        ConfigOptions.key(READER_PREFIX + "content_type")
+        key(READER_PREFIX + "content_type")
+            .noDefaultValue(String.class);
+  }
+
+  interface BaseFileReaderOptions extends BaseReaderOptions {
```

Comment:
Options in BaseFileReaderOptions are never been used. Can we remove BaseFileReaderOptions?

Reply:
It seems some of CSV/JSON options are needed by hadoop and ftp connector...

Reply:
Never mind, found those in RowParserOptions, will remove~